_data/simple-icons.json

```
@@ -10408,8 +10408,8 @@
 	{
 		"title": "Mozilla",
 		"hex": "000000",
```

Mozilla list their primary colour as 'off-black'.

```suggestion
		"hex": "161616",
```